src/main/scala/uk/gov/nationalarchives/tdr/api/service/FullHealthCheckService.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.nationalarchives.tdr.api.service
+
+import slick.jdbc.PostgresProfile.api._
+import uk.gov.nationalarchives.tdr.api.db.DbConnection
+import uk.gov.nationalarchives.tdr.api.db.repository.TransferringBodyRepository
+
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Instead of using the global execution context, can you pass it into the service constructor.